tests/ClientTest.php

```diff
@@ -332,24 +332,27 @@ public function testAddConnectionWithConcreteWriteStream()
 
     /**
      * Tests that adding a connection configured to use the SSL transport and
-     * force use of IPv4 throws an exception.
+     * force use of IPv4 does not throw an exception.
      *
      * @see https://github.com/reactphp/socket-client/issues/4
      */
-    public function testAddConnectionWithSslTransportAndForceIpv4ThrowsException()
+/*
+    public function testAddConnectionWithSslTransportAndForceIpv4()
     {
         $connection = $this->getMockConnectionForAddConnection();
         Phake::when($connection)->getOption('transport')->thenReturn('ssl');
         Phake::when($connection)->getOption('force-ipv4')->thenReturn(true);
+        $writeStream = $this->getMockWriteStream();
+        Phake::when($this->client)->getWriteStream($connection)->thenReturn($writeStream);
+
+        $this->client->setLogger($this->getMockLogger());
         $this->client->setResolver($this->getMockResolver());
+        $this->client->addConnection($connection);
 
-        try {
-            $this->client->addConnection($connection);
-            $this->fail('Expected exception was not thrown');
-        } catch (Exception $e) {
-            $this->assertSame(Exception::ERR_CONNECTION_STATE_UNSUPPORTED, $e->getCode());
-        }
+        print_r(stream_context_get_options($connection->getData('stream')));
+        //print_r($actual);
```

Comment:
[I commented about this earlier](https://github.com/phergie/phergie-irc-client-react/pull/58#discussion_r99470640).